wavefront/server/modules/user_management_module/user_management_module/controllers/access_controller.py

```
@@ -221,20 +203,39 @@ async def get_resource(
         Provide[CommonContainer.response_formatter]
     ),
     user_service: UserService = Depends(Provide[UserContainer.user_service]),
-    scopes: list[str] = Query(
-        default=[ResourceScope.DASHBOARD, ResourceScope.CONSOLE],
-        description='The scopes of the resources to fetch',
+    scopes: Optional[list[str]] = Query(
+        default=None,
+        description='Scopes of the resources to fetch (all scopes when omitted)',
+    ),
```

Ruff (0.15.15)

[warning] Do not perform function call `Query` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)